webpack/assets/javascripts/react_app/components/about/compute/schema.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import ConnectedStatus from '../../common/status';
+import { headerFormat, cellFormat } from '../../common/table';
+import helpers from '../../../common/helpers';
+
+export const columns = [
```

Comment:
Any thoughts to how this will work with selectable columns?

Reply:
It should work,
we can follow patternfly-react [example](https://github.com/patternfly/patternfly-react/blob/master/src/components/Table/__mocks__/mockClientPaginationTable.js)